pkg/vulnerabilityreport/controller/workload.go

```diff
@@ -94,6 +94,10 @@ func (r *WorkloadController) SetupWithManager(mgr ctrl.Manager) error {
 	// Determine which Kubernetes workloads the controller will reconcile and add them to resources
 	targetWorkloads := r.Config.GetTargetWorkloads()
```

Comment:
I believe we also should update `GetTargetWorkloads()` as well https://github.com/aquasecurity/trivy-operator/blob/6cf2d176262995ee777fa5c53ff88695393882bd/pkg/operator/etc/config.go#L133

Reply:
updated https://github.com/aquasecurity/trivy-operator/pull/2693/commits/445eae3f398c1d4f594b8f7c1c6545700188988b